app/controllers/admin_controller.rb

```diff
@@ -0,0 +1,25 @@
+class AdminController < ActionController::Base
```

Comment:
So when it comes to making admin type directories we will want to do it in such a way that we can both stay RESTful and can separate functionality from other controllers. So for the admin one, you will probably want to create a directory called `admin` and inside of that directory have a file called `shelters_controller.rb` and possibly `applications_controller.rb`. Now you can remain RESTful with CRUD actions. The class would look like:
```ruby
class Admin::SheltersController < ApplicationController

end
```
Now you can have an index action that is essentially doing what your `shelters_index` action is doing.